app/inflation/README.md

```diff
@@ -0,0 +1,97 @@
+
+# UMEE Tokenomic Restructuring Proposal
+
+## Introduction
+
+The UMEE Tokenomic Restructuring proposal aims to optimize the tokenomics of the $UMEE token, which is issued following the standard Cosmos SDK mechanism. The goal is to progressively limit the token issuance in an algorithmic way, taking into account various factors such as network security, adoption of liquid staking, and additional income sources for stakers through interchain security and chain-based revenue streams.
+
+## Design Overview
+
+1. The $UMEE 2.0 Tokenomics will adopt a similar Cosmos dynamic inflation mechanism based on the bonding rate of the network. This allows the Umee chain to naturally adjust the inflation rate to drive staking activities and reach the targeted bonding rate.
+
+2. To initiate the new tokenomics, the maximum and minimum inflation rates will decrease by 25%. Subsequent decreases will occur every two years.
```

Comment:
```suggestion
2. When these tokenomics changes take effect, the maximum and minimum inflation rates will immediately decrease by 25%. Subsequent decreases will occur every two years.
```

Reply:
Before this new token emission , we will reduce 20% of the current min and max inflation rate by Gov Proposal 

I will update this section